accounts/src/main/java/eu/kevin/accounts/accountsession/AccountSession.kt

```diff
@@ -102,10 +104,15 @@ internal class AccountSession(
 
     private fun updateFlowItems() {
         val flow = mutableListOf<AccountSessionFlowItem>()
-        if (!configuration.skipBankSelection || sessionData.selectedBank == null) {
-            flow.add(BANK_SELECTION)
+
+        if (sessionData.linkingType == AccountLinkingType.BANK) {
```

Comment:
nested if could be simplified

Reply:
Extracted into separate method, I think it makes `updateFlowItems` more readable